src/components/layout.jsx

```diff
@@ -91,4 +93,4 @@ const Layout = ({ children }) => (
   </ThemeProvider>
 )
 
-export default Layout
+export default errorHandler(Layout)
```

Comment:
I think it make sense to wrap `children` passed to this component into ErrorHandler and do not replace the whole app with error message, but only fallen component (you can convert ErrorHandler to ordinary react component). In this case we'll be able to catch errors more granularly and user will be able to continue using site (e.g. to go to another page)

Reply:
Some improvements:
* the error splash replaces the children component only;
* ReactDOM.render is used to display a global error splash in the case of application fully crash.